gridcapacity/capacity_analysis.py

```diff
@@ -20,7 +20,7 @@
 from collections import OrderedDict, defaultdict
 from collections.abc import Collection
 from dataclasses import dataclass
-from typing import Final, Generator, Iterator, Optional, Tuple
+from typing import Any, Final, Generator, Iterator, Optional, List
```

Comment:
The `typing.List` [is deprecated](https://docs.python.org/3.11/library/typing.html#typing.List).

Reply:
we target python 3.9 still, I suppose due to psse api compatibility mostly. [There it is not](https://docs.python.org/3.9/library/typing.html#typing.List)

But will keep in mind for next iterations, thanks!